### PR TITLE
Initial template override support

### PR DESCRIPTION
**Pull request by rtfpessoa**

### CODE REVIEW

File: src/file-list-printer.js [hunk 1]
```diff
@@ -8,11 +8,16 @@
...
   var printerUtils = require('./printer-utils.js').PrinterUtils;
 
-  var hoganUtils = require('./hoganjs-utils.js').HoganJsUtils;
+  var hoganUtils;
+
   var baseTemplatesPath = 'file-summary';
   var iconsBaseTemplatesPath = 'icon';
 
-  function FileListPrinter() {
+  function FileListPrinter(config) {
```
Comment (zhihuo) [inline src/file-list-printer.js]: ttttt

File: src/file-list-printer.js [hunk 2]
```diff
@@ -8,11 +8,16 @@
 (function() {
   var printerUtils = require('./printer-utils.js').PrinterUtils;
 
-  var hoganUtils = require('./hoganjs-utils.js').HoganJsUtils;
+  var hoganUtils;
+
   var baseTemplatesPath = 'file-summary';
   var iconsBaseTemplatesPath = 'icon';
```
Comment (zhihuo) [inline src/file-list-printer.js]: how to use it

**Comment by rtfpessoa** [comment]
Hey, check https://github.com/rtfpessoa/diff2html#configuration for templates and rawTemplates.

File: src/file-list-printer.js (hunk 2 above)
Comment (rtfpessoa) [inline src/file-list-printer.js]: Hey, check https://github.com/rtfpessoa/diff2html#configuration for templates and rawTemplates.

**Comment by jili0503** [comment]
test，test

**Comment by ysyyrps** [comment]
dddddd

**Pull request merged**